.github/workflows/build.yml

```diff
@@ -71,7 +71,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: ["1.22"]
+        go-version: ["1.22", "1.23]
```

Comment:
should we change the CI??

Reply:
I was not sure. We could do it as follow-up.